### ISSUE TITLE
[Bug] WinMerge: Wrong architecture installer downloaded

### ISSUE THREAD

**Issue by sdottaka** [tunisiano187/Chocolatey-packages#3001]
**Describe the bug**

This WinMerge Chocolatey package downloads installers with improper architecture starting with WinMerge 2.16.12.

The installers for each architecture have the following filenames:

32-bit Installer: WinMerge-2.16.13-Setup.exe
64-bit Installer: WinMerge-2.16.13-x64-Setup.exe
ARM64 Installer: WinMerge-2.16.13-ARM64-Setup.exe

The download URL of the 64-bit version installer in this package is the 32-bit version installer(WinMerge-2.16.13-Setup.exe), and the download URL of the 32-bit version installer is the ARM64 version installer (WinMerge-2.16.13-ARM64-Setup.exe).

Probably due to the addition of the ARM64 version of the installer from WinMerge 2.16.12.

I posted the following Pull request, but I couldn't resolve the AppVeyorBot "Package verification failed", so I withdrew the Pull request and post this issue.

https://github.com/tunisiano187/Chocolatey-packages/pull/3000

**To Reproduce**
1. Run `choco install winmerge -pre` on Windows 10 64bit version

~~~
c:\>choco install winmerge -pre
Chocolatey v0.10.15
[Pending] Removing incomplete install for 'winmerge'
Installing the following packages:
winmerge
By installing you accept licenses for the packages.
Progress: Downloading winmerge 2.16.13-pre20210622... 100%

winmerge v2.16.13-pre20210622
winmerge package files install completed. Performing other installation steps.
The package winmerge wants to run 'chocolateyInstall.ps1'.
Note: If you don't run this script, the installation will fail.
Note: To confirm automatically next time, use '-y' or consider:
choco feature enable -n allowGlobalConfirmation
Do you want to run the script?([Y]es/[A]ll - yes to all/[N]o/[P]rint): y

Downloading WinMerge 64 bit
  from 'https://github.com/WinMerge/winmerge/releases/download/v2.16.13/WinMerge-2.16.13-Setup.exe'
Progress: 100% - Completed download of C:\Users\sawan\AppData\Local\Temp\chocolatey\winmerge\2.16.13-pre20210622\WinMerge-2.16.13-Setup.exe (7.13 MB).
Download of WinMerge-2.16.13-Setup.exe (7.13 MB) completed.
Hashes match.
Installing WinMerge...
WinMerge has been installed.
  winmerge can be automatically uninstalled.
Environment Vars (like PATH) have changed. Close/reopen your shell to
 see the changes (or in powershell/cmd.exe just type `refreshenv`).
 The install of winmerge was successful.
  Software installed to 'C:\Program Files (x86)\WinMerge\'

Chocolatey installed 1/1 packages.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

c:\>
~~~

**Expected behavior**

Install the 64-bit version of WinMerge when OS is 64-bit versions of Windows

**Screenshots**

![image](https://user-images.githubusercontent.com/98126/125142087-db227080-e151-11eb-87f4-b346eb90477a.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version v2.16.13-pre20210622

**Additional context**

**Comment by jwfxpr** [comment]
That may explain why the 2.16.12 package has been [stuck in moderation](https://community.chocolatey.org/packages/winmerge/2.16.12) since May.

**Comment by jwfxpr** [comment]
@tunisiano187 can you confirm if the version issues from the 2.16.12 package have been resolved in [2.16.14](https://community.chocolatey.org/packages/winmerge/2.16.14) that was submitted to chocolatey on 26 July?
